libs/core/langchain_core/prompts/chat.py

```diff
@@ -1053,9 +1053,9 @@ def validate_input_variables(cls, values: dict) -> Any:
                 if message.variable_name not in input_types:
                     input_types[message.variable_name] = list[AnyMessage]
         if "partial_variables" in values:
-            input_vars = input_vars - set(values["partial_variables"])
+            input_vars -= set(values["partial_variables"])
```

Comment:
Is this an auto fix as well? Why the change?

Reply:
Yes it’s an auto-fix